File: src/it/nominasuntsubstantiarerum/netbus/entity/EntityAutobus.java
package it.nominasuntsubstantiarerum.netbus.entity;

public class EntityAutobus {
	private int idAutobus;
	private int capienza;
	
	public EntityAutobus(int idAutobus, int capienza) {
		this.idAutobus = idAutobus;
		this.capienza = capienza;
	}
	
	public int getIdAutobus() {
		return idAutobus;
	}
	
	public void setIdAutobus(int idAutobus) {
		this.idAutobus = idAutobus;
	}
	
	public int getCapienza() {
		return capienza;
	}
	
	public void setCapienza(int capienza) {
		this.capienza = capienza;
	}
}

File: htdocs/entity/EntityAutobus.php
<?php

class EntityAutobus {
    private int $idAutobus;
    private int $capienza;

    public function __construct(int $idAutobus, String $capienza) {
        $this->idAutobus = $idAutobus;
        $this->capienza = $capienza;
    }

    // Getter e Setter per idAutobus
    public function getIdAutobus(): int {
        return $this->idAutobus;
    }

    public function setIdAutobus(int $idAutobus): void {
        $this->idAutobus = $idAutobus;
    }

    // Getter e Setter per capienza
    public function getCapienza(): int {
        return $this->capienza;
    }

    public function setCapienza(int $capienza): void {
        $this->capienza = $capienza;
    }
}
?>